.changeset/modern-ghosts-sin.md

```diff
@@ -0,0 +1,41 @@
+---
+"@logto/tunnel": minor
+---
+
+add deploy command and env support
+
+#### Add new `deploy` command to deploy your local custom UI assets to your Logto Cloud tenant
+
+1. Create a machine-to-machine app in your Logto tenant
```

Comment:
Need to assign related role(with management api access) to this m2m app.

Reply:
Improved